components/CraftsList/CraftsList.tsx

```diff
@@ -156,7 +160,7 @@ export function CraftsList(props: Props) {
     }
 
     function getListElement(craft: ProfitableCraft, blur: boolean) {
-        if (nameFilter && craft.item.name?.toLowerCase().indexOf(nameFilter.toLowerCase()) === -1) {
+        if (((nameFilter && craft.item.name?.toLowerCase().indexOf(nameFilter.toLowerCase()) === -1) || craft.craftCost < minimumProfit)&& !blur ) {
```

Comment:
Shouldnt the profit be "craftingCost - sellPrice"?
Currently it is a "minimum craftCost" filter

Reply:
Also: Isnt the check useless here? As it is already filtered below?

Reply:
wait yeah you're right